### PR TITLE
Setup views

### PR DESCRIPTION
**Pull request by shasherazi**
In this PR, I added views for the following paths:
- index
- /users
- /users/:user_id
- /users/:user_id/posts
- /users/:user_id/posts/:id

according the wireframes provided.

To run the project, you have to load the sample data, which you can load using the following command:
```bash
rake db:populate
```

### CODE REVIEW

File: app/assets/stylesheets/users.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  padding: 0 10px;
+  margin-bottom: 20px;
+}
+
+.user-posts-preview {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
```
Comment (rloterh) [inline app/assets/stylesheets/users.css]: - [x] Awesome job! Your views are working great! However, let's add a few tweaks to ensure that the design complies with the wireframe provided; Let's ensure that the user picture is not joined in one border with the user bio.

**Your designs:**
**View 1**        |  **View 2**        |  **View 3**
:-------------------------:|:-------------------------:|:-------------------------:
Your desgin |  Home Page | Home Page

**Wireframe designs:**
**View 1**        |  **View 2**        |  **View 3**
:-------------------------:|:-------------------------:|:-------------------------:
Your desgin |  Home Page | Home Page

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Shasherazi, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!
🚦 _ 🏡 🏃‍♂️ 🏚️ 🌴 🗼 

## Highlights 👍  
- User details and post views properly added!
- ReadMe looks very professional!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/assets/stylesheets/users.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply app/assets/stylesheets/users.css]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @shasherazi 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**